doc/source/whatsnew/v0.19.0.txt

```diff
@@ -755,3 +755,4 @@ Bug Fixes
 - Bug where ``pd.read_gbq()`` could throw ``ImportError: No module named discovery`` as a result of a naming conflict with another python package called apiclient  (:issue:`13454`)
 - Bug in ``Index.union`` returns an incorrect result with a named empty index (:issue:`13432`)
 - Bugs in ``Index.difference`` and ``DataFrame.join`` raise in Python3 when using mixed-integer indexes (:issue:`13432`, :issue:`12814`)
+- Bug in ``pd.Timedelta(None)`` raises ``ValueError``. This is different from ``pd.Timestamp(None)`` (:issue:`13687`)
```

Comment:
How about:

- Bug in ``pd.Timedelta(None)`` raises ``ValueError`` rather than returning ``pd.NaT`` as the same as ``pd.Timestamp(None)``

pls wait for @jreback and @jorisvandenbossche comments as i'm not a native:)